apps/src/util/unsupportedBrowserWarning.js

```diff
@@ -17,10 +17,7 @@ export var checkForUnsupportedBrowsersOnLoad = function() {
       if (isMobileDevice()) {
         if (appOptions.app === 'applab') {
           textDivId = '#applab-unsupported-tablet';
-        } else if (
-          appOptions.app === 'gamelab' &&
-          !appOptions.valueTypeTabShapeMap
```

Comment:
What was the reason for removing this?

Reply:
The goal here was to show the warning for gamelab but not spritelab (`valueTypeTabeShapeMap` is only set for spritelab). Now that spritelab is its own app type, though, this check can just be `appOptions.app === 'gamelab'`

Reply:
ugh... valueTypeTabShapeMap identifying spritelab. 

Thanks for catching that!